lib/slack.ml

```diff
@@ -413,7 +437,15 @@ let generate_status_notification ~(job_log : (string * string) list) ~(cfg : Con
         | Some pipeline_url -> sprintf "<%s|[%s]>: %s for \"%s\"" pipeline_url context build_desc commit_message)
   in
   let text = Some (String.concat "\n" @@ List.concat [ commit_info; branches_info ]) in
-  let attachment = { empty_attachments with mrkdwn_in = Some [ "fields"; "text" ]; color = Some color_info; text } in
+  let attachment =
+    {
+      empty_attachments with
+      mrkdwn_in = Some [ "fields"; "text" ];
+      color = Some color_info;
+      text;
+      fields = Option.map (fun a -> [ a ]) job_log_peek;
```

Comment:
no need to `Option.map` if you add the list in the `job_log_peek` call.

also: odd var name for the argument

Reply:
changed both